### PR TITLE
Correctly document MAC address generation

### PR DESCRIPTION
**Pull request by gianlucaborello**

### CODE REVIEW

File: docs/sources/reference/run.md [hunk 1]
```diff
@@ -216,9 +216,9 @@ networking. In cases like this, you would perform I/O through files or
 Your container will use the same DNS servers as the host by default, but
 you can override this with `--dns`.
 
-By default a random MAC is generated. You can set the container's MAC address
-explicitly by providing a MAC via the `--mac-address` parameter (format:
-`12:34:56:78:9a:bc`).
+By default, the MAC is generated using the IP address allocated to the container.
```
Comment (thaJeztah) [inline docs/sources/reference/run.md]: the MAC *address*

**Comment by thaJeztah** [comment]
Thanks, LGTM, one small nit

**Comment by thaJeztah** [comment]
Moved to docs review as I don't think this requires code review here.

Windows build failed, which probably needs to be given a kick

cc @fredlf @jamtur01 @moxiegirl @SvenDowideit

**Comment by jamtur01** [comment]
Docs LGTM

**Comment by thaJeztah** [comment]
@gianlucaborello if you could address my minor comment and amend your commit, then I think we're ready to merge (if the builds succeed this time)

**Comment by gianlucaborello** [comment]
@thaJeztah updated, thanks

**Comment by thaJeztah** [comment]
Thanks for your contribution, @gianlucaborello!

**Pull request merged**